port/unix/omrsysinfo.c

```diff
@@ -4465,70 +4446,108 @@ omrsysinfo_cgroup_subsystem_iterator_hasNext(struct OMRPortLibrary *portLibrary,
 	return check;
 }
 
+int32_t
+omrsysinfo_cgroup_subsystem_iterator_metric_key(struct OMRPortLibrary *portLibrary, const struct OMRCgroupMetricIteratorState *state, const char **metricKey)
+{
+	int32_t rc = OMRPORT_ERROR_SYSINFO_CGROUP_SUBSYSTEM_METRIC_NOT_AVAILABLE;
+#if defined(LINUX) && !defined(OMRZTPF)
```

Comment:
Code should validate that `metricKey` is not null.

Reply:
added validation